index.js

```diff
@@ -43,6 +43,30 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Installation
+//.
+//. Sanctuary can be installed via npm:
+//.
+//. ```console
+//. $ npm install sanctuary
+//. ```
+//.
+//. Or simply include a script tag in your html document:
+//.
+//. ```html
+//. <script src="https://unpkg.com/sanctuary@latest/dist/sanctuary.min.js"></script>
+//. ```
+//.
+//. You can also try Sanctuary in your browser right now in the
+//. [Ramda REPL](https://goo.gl/pUU9Ye).
```

Comment:
Please (temporarily) add this snippet to the pull request to make it easier to review. :)

Reply:
:zap: